queries/astro/highlights.scm

```diff
@@ -19,3 +19,9 @@
 ((erroneous_end_tag
   (erroneous_end_tag_name) @type)
   (#lua-match? @type "^[A-Z]"))
+
+(start_tag
+  (fragment_tag_name) @tag.delimiter)
```

Comment:
tag.delimiter is for `<`, `>` and the like, and fragment is, I'd assume, `</>`? So naming it as tag_name and then capturing with tag.delimiter seems a bit off